examples/daal/cpp_sycl/source/datasource/row_merged_to_sycl_homogen.cpp

```diff
@@ -0,0 +1,155 @@
+/* file: datastructures_usm.cpp */
+/*******************************************************************************
+ * Copyright 2021 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+/*
+!  Content:
+!    Example of the use of Unified Shared Memory in Row Merged Numeric Table
+!
+!******************************************************************************/
+
+/**
+ * <a name="DAAL-EXAMPLE-CPP-ROWMERGEDTOSYCLHOMOGEN"></a>
+ * \example datastructures_usm.cpp
+ */
+
+#include "daal_sycl.h"
+#include "service_sycl.h"
+
+using namespace daal;
+using namespace daal::services;
+using namespace daal::data_management;
+
+using daal::data_management::internal::SyclHomogenNumericTable;
+using daal::services::internal::SyclExecutionContext;
+
+#if (defined(__SYCL_COMPILER_VERSION)) && (__SYCL_COMPILER_VERSION >= 20191001)
+#define IS_USM_SUPPORTED
+#endif
```

Comment:
I think, we need to remove this condition, since all modern compiler versions supports USM

Reply:
Done